samples/server.js

```diff
@@ -22,9 +22,7 @@ const express = require('express');
 const app = express();
 app.enable('trust proxy');
 
-app.use(bodyParser.raw());
-app.use(bodyParser.json());
-app.use(bodyParser.text());
+app.use(bodyParser.raw({type: 'application/octet-stream'}));
```

Comment:
This `octet-stream` line could use a comment. Why are we using it instead of raw/json/text?

Reply:
Added.